docs/API.md

```diff
@@ -646,55 +646,22 @@ for (Result<Item> result : results) {
 ```
 
 <a name="makeBucket"></a>
-### makeBucket(String bucketName)
-`public void makeBucket(String bucketName)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#makeBucket-java.lang.String-)_
+### makeBucket(MakeBucketArgs args)
+`public void makeBucket(MakeBucketArgs args)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#makeBucket-io.minio.messages.MakeBucketArgs-)_
 
-Creates a bucket with default region.
+Creates a bucket with given region and object lock feature enabled.
 
 __Parameters__
-| Parameter      | Type     | Description         |
-|:---------------|:---------|:--------------------|
-| ``bucketName`` | _String_ | Name of the bucket. |
+| Parameter      | Type             | Description                |
+|:---------------|:-----------------|:---------------------------|
+| ``args      `` | _MakeBucketArgs_ | Arguments to create bucket |
 
 __Example__
 ```java
-minioClient.makeBucket("my-bucketname");
+MakeBucketArgs args = MakeBucketArgs.newBuilder().bucket("mybucket").region("us-east-1").objectLock(true).build();
```

Comment:
1. This example code should be spotless checked.
2. Provide multiple examples like
```
// Create bucket with default region.
minioClient.makeBucket(MakeBucketArgs.newBuilder().bucket("my-bucketname").build());

// Create bucket with specific region.
....

// Create object-lock enabled bucket with specific region.
...
```